src/main/java/Tasks.java

```diff
@@ -0,0 +1,38 @@
+public class Tasks {
+    protected String TaskName;
+    protected boolean isDone;
+    protected static int NumberOfTasks;
```

Comment:
Please take note of the naming convention for variables; they should be in camelCase.